test/functional/change_stream.test.js

```diff
@@ -786,7 +793,7 @@ describe('Change Streams', function() {
     }
   });
 
-  it('Should return MongoNetworkError after first retry attempt fails using promises', {
+  it.skip('Should return MongoNetworkError after first retry attempt fails using promises', {
```

Comment:
These two`Should return MongoNetworkError after first retry attempt fails` tests started timing out now that `.hasNext` resumes from errors. They just keep resuming from the network errors; none of the resume attempts fail. I'm guessing these tests are quite old and it might make more sense to remove them than to try to fix them.